scripts/circle-setup

```diff
@@ -116,15 +116,10 @@ install_cni_plugins() {
 }
 
 install_runc() {
-    export GOPATH=/home/circleci/go
-    mkdir -p "$GOPATH"/github.com/opencontainers
-    RUNC_PATH=github.com/opencontainers/runc
-    git clone https://"$RUNC_PATH" "$GOPATH"/src/"$RUNC_PATH"
-    pushd "$GOPATH"/src/$RUNC_PATH
-    git checkout "${VERSIONS["runc"]}"
-    make BUILDTAGS="apparmor selinux seccomp"
-    sudo cp runc /usr/sbin
-    popd
+    URL=https://github.com/opencontainers/runc/releases/download
+    BINARY=/usr/sbin/runc
+    sudo wget -O "$BINARY" $URL/"${VERSIONS["runc"]}"/runc.amd64
+    sudo chmod +x "$BINARY"
     runc --version
```

Comment:
should you replace runc with $BINARY here?

Reply:
Hm, no this is more just a verification the `runc` is in `$PATH` :)

Reply:
For security reasons, I would suggest compare sha256sum from the file before using it.

$ wget https://github.com/opencontainers/runc/releases/download/v1.0.0-rc10/runc.amd64
$ wget https://github.com/opencontainers/runc/releases/download/v1.0.0-rc10/runc.sha256sum

$ sha256sum runc.amd64 
a01afd5ff47d5a2a96bea3a871fb445b432f90c249a8a5d5239b05fe0d5bee4a  runc.amd64

$ cat runc.sha256sum | grep -w a01afd5ff47d5a2a96bea3a871fb445b432f90c249a8a5d5239b05fe0d5bee4a

a01afd5ff47d5a2a96bea3a871fb445b432f90c249a8a5d5239b05fe0d5bee4a  runc.amd64

Reply:
Yes, I add this as well.